Makefile

```diff
@@ -266,6 +266,7 @@ generate-go: ## Generate golang sources
 		paths=github.com/vmware-tanzu/vm-operator/api/... \
 		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt
 	$(MAKE) -C ./pkg/util/cloudinit/schema $@
+	go generate ./api/v1alpha3
```

Comment:
Should we put this before running the `controller-gen` command above?

Reply:
Probably :) Thanks @dilyar85 !